src/main/javac/org/apache/brooklyn/tosca/a4c/platform/transformer/ToscaPlanToSpecTransformer.java

```diff
@@ -258,6 +296,17 @@ private static boolean isTosca(Map<?,?> obj) {
             }
         }
 
+        //process child request
```

Comment:
Once the nodeTemplates were processed, the children can be added to their parents